main/zoe/api/zoe-contract-facet.md

```diff
@@ -4,23 +4,20 @@
 
 A Zoe Contract Facet is an API object for a running contract instance to access the Zoe state for that instance. A Zoe Contract Facet is accessed synchronously from within the contract, and usually is referred to in code as `zcf`. The contract instance is launched by `E(zoe).makeInstance`, and is given access to the `zcf` object during that launch. In the operation below, the `instanceHandle` is the handle for the running contract instance.
```

Comment:
That's an awfully complicated first sentence. Suggest a change to "A Zoe Contract Facet is an API object for a running contract instance. It accesses the instance's Zoe state." I'd also change "usually is" to "is usually". Change the third sentence to "The contract instance is launched and given access to the `zcf` object by `E(zoe).makeInstance`." Finally, "Below, `instanceHandle` is the running contract instance's handle."